libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -697,7 +697,7 @@ dictionary SendSpontaneousPaymentRequest {
 };
 
 dictionary SendPaymentResponse {
-    Payment payment;
+    PaymentListItem payment;
```

Comment:
This should probably contain the PendingPayment

Reply:
I think both here and in PaymentSucceed event we should contain the same struct/data.
@JssDWt @hydra-yse  how about using the LNPaymentDetails here and there? This way we don't need to introduce another struct (PendingPayment).

Reply:
Question: How would you feel about extending the PaymentDetails enum with variants for lnurl_pay, ln_address, lnurl_withdraw, swap and reverse swap? And make LnPaymentDetails basically the thing we mean with PendingPayment here?